app/views/research_sessions/preview.html.erb

```diff
@@ -76,13 +76,13 @@
   <section>
     <h3 class="subtitle-small" id="doi">Do I have to take part?</h3>
     <p>
-      Taking part is entirely voluntary - it is up to you to decide whether or not
-      to take part. If you decide to take part you do not have to answer questions
-      you do not want to answer. You can also change your mind about taking part
+      Taking part is entirely voluntary - it is up to you to decide whether or not <%= @research_session.able_to_consent? ? 'you' : 'your child/the child in your care' %>
+      to take part. If you decide <%= @research_session.able_to_consent? ? 'you' : 'your child/the child in your care' %> to take part <%= @research_session.able_to_consent? ? 'you' : 'they' %> do not have to answer questions
+      <%= @research_session.able_to_consent? ? 'you' : 'they' %> do not want to answer. <%= @research_session.able_to_consent? ? 'You' : 'They' %> can also change <%= @research_session.able_to_consent? ? 'your' : 'their' %> mind about taking part
       at any time and withdraw without giving a reason.
     </p>
     <p>
-      If you agree to take part, you will be given a copy of this information
+      If you agree <%= 'for your child\'s/the child\'s in your care' if @research_session.unable_to_consent? %> to take part, you will be given a copy of this information
```

Comment:
We need to remove the `'s`s here, otherwise we get "If you agree for your child's/the child's in your care to take part"